CMakeLists.txt

```diff
@@ -22,6 +22,7 @@ target_include_directories(${ESP_STUB_LIB}
     PRIVATE include/esp-stub-lib
 )
 
+message("DBG ESP_STUB_LIB: ${ESP_STUB_LIB}, STUB_COMPILE_DEFS: ${STUB_COMPILE_DEFS}")
```

Comment:
Can we remove the log here? When we need to see all flags, we can see them with `cmake --build . --verbose`  inside build/<target> directory